src/styles/blocks/todolist.scss

```diff
@@ -0,0 +1,143 @@
+.todolist {
+  margin: 0;
+  padding: 0;
+
+  list-style: none;
+
+  &__item {
+    position: relative;
+
+    border-bottom: 1px solid #ededed;
+
+    font-size: 24px;
+
+    &:last-child {
+      border-bottom: none;
+    }
+
+    &--editing {
+      padding: 0;
+
+      border-bottom: none;
+
+      &:last-child {
+        margin-bottom: -1px;
+      }
+    }
+
+    &--completed label {
+      color: #d9d9d9;
+      text-decoration: line-through;
+    }
+
+    & label {
+      display: block;
+
+      padding: 15px 15px 15px 60px;
+      transition: color 0.4s;
+
+      word-break: break-all;
+      line-height: 1.2;
+    }
+  }
+
+  &__edit-field {
+    display: none;
+
+    &:focus {
+      outline: 0;
+    }
+  }
+
+  &__item--editing &__edit-field {
+    display: block;
+
+    padding: 12px 16px;
+    margin: 0 0 0 $editFieldeftMargin;
+    width: 467px;
```

Comment:
Replace the '$editFieldeftMargin' with a valid SCSS variable, it seems to be a typo.

Reply:
Fixed.